README.md

```diff
@@ -1,2 +1,44 @@
 # HowToCar
 Codefellows_301_Final_Project
```

Comment:
Nice work on documenting the user stories. It's worthwhile to take this as a stepping point, and make sure that documenting your projects is part of your typical workflow. It should also be done in a formal manner, which you can see by looking at well know open-source API's, like NodeJS or HandlebarsJS. Check out some examples; they all follow a pretty common format.